programs/server/config.xml

```diff
@@ -1386,4 +1386,10 @@
         <lru_cache_size>268435456</lru_cache_size>
         <continue_if_corrupted>true</continue_if_corrupted>
     </merge_tree_metadata_cache-->
+
+    <!-- This allows to disable exposing addresses in stack traces for security reasons.
+         Please be aware that it does not improve security much, but makes debugging much harder.
+         I don't recommend to change this setting.
+    -->
+    <show_addresses_in_stack_traces>true</show_addresses_in_stack_traces>
```

Comment:
Should we add a note about system tables like `stack_traces` and `trace_log` (that this setting use useless without restricting access to system tables)?